search_algorithms/missing_number.py

```diff
@@ -0,0 +1,11 @@
+def missing_number(missing_Arr):
+    i = 0
+    while i < len(missing_Arr):
+        if i != missing_Arr[i]:
+            return i
+        i += 1
+
+
+arr = [0,1,2,3,4,5,6,8,9, 10]
```

Comment:
```suggestion
arr = [0,1,2,3,4,5,6,8,9, 10]
```
Remember delete this for next refactor

Reply:
And all test case without unit test

Reply:
In the next sprint